docs/en/week07/07-3.md

```diff
@@ -0,0 +1,133 @@
+---
+lang-ref: ch.07-3
+title: Introduction to autoencoders
+lecturer: Alfredo Canziani
+authors: Vidit Bhargava, Monika Dagar
+date: 18 March 2021
+---
+# DALL-E: Creating Images from Text
+
+[![hackmd-github-sync-badge](https://hackmd.io/usp9PiUGRdGQzD93wE8Bsw/badge)](https://hackmd.io/usp9PiUGRdGQzD93wE8Bsw)
+
+A neural network that creates images from text captions. 
+![](https://i.imgur.com/FosonFM.jpg)
+
+
+Go to the [website](https://openai.com/blog/dall-e/) and play with the captions! 
+
+# Autoencoder
+Let's start with some definitions:
+
+## Definitions
+### Input
+
+$\boldsymbol{x}$: is observed during both training and testing 
```

Comment:
I said to use `\vect{}` for vectors. It's a macro, so we handle how all vectors appear.
Similarly, use `\matr{}` for matrices.
Do read [the instructions](https://atcold.github.io/pytorch-Deep-Learning/en/about/) please.